Sources/HTTP/Response/Response.swift

```diff
@@ -0,0 +1,83 @@
+import Transport
+
+// So common we simplify it
+
+import URI
+
+public final class Response: Message {
+    public var version: Version
```

Comment:
same variability question here, I think some more constants or internal settables might be better.